src/test/java/com/datadog/api/v2/client/api/LogsApiTest.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache-2.0 License.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+package com.datadog.api.v2.client.api;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.datadog.api.RecordingMode;
+import com.datadog.api.TestUtils;
+import com.datadog.api.v2.client.ApiException;
+import com.datadog.api.v2.client.model.LogsListRequest;
+import com.datadog.api.v2.client.model.LogsListRequestFilter;
+import com.datadog.api.v2.client.model.LogsListRequestPage;
+import com.datadog.api.v2.client.model.LogsListResponse;
+import com.datadog.api.v2.client.model.LogsSort;
+import java.net.MalformedURLException;
+import java.net.URLEncoder;
+import java.time.Duration;
+import javax.ws.rs.core.GenericType;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+@SuppressWarnings("ConstantConditions")
+public class LogsApiTest extends V2APITest {
+
+    private static LogsApi api;
+
+    @Override
+    public String getTracingEndpoint() {
+        return "logs";
+    }
+
+    @BeforeClass
+    public static void enableUnstableOperations() {
+        generalApiClient.setUnstableOperationEnabled("listLogs", true);
+        generalApiClient.setUnstableOperationEnabled("listLogsGet", true);
+    }
+
+    @AfterClass
+    public static void disableUnstableOperations() {
+        generalApiClient.setUnstableOperationEnabled("listLogs", false);
+        generalApiClient.setUnstableOperationEnabled("listLogsGet", false);
+    }
+
+    @Before
+    public void setup() {
+        api = new LogsApi(generalApiClient);
+    }
+
+    private void sendLogs(String suffix) throws ApiException, MalformedURLException {
+        long nowNano = now.toEpochSecond() * 1000000 + now.getNano();
```

Comment:
It would be nice to use `getUniqueEnitityName()`

https://github.com/DataDog/datadog-api-client-java/blob/master/src/test/java/com/datadog/api/TestUtils.java#L380